src/components/Register.jsx

```diff
@@ -53,6 +62,43 @@ const Register = ({ toggleForm }) => {
       return;
     }
 
+    const hasUpperCase = /[A-Z]/.test(values.password);
+    const hasSpecialChar = /[!@#$%^&*(),.?":{}|<>]/.test(values.password);
+    let missingPasswordRequirements = [];
+
+    if (!hasUpperCase) {
+      missingPasswordRequirements.push(<li key="upper">Capital letter</li>);
+    }
+    if (!hasSpecialChar) {
+      missingPasswordRequirements.push(
+        <li key="special">Special character</li>
+      );
+    }
+
+    if (missingPasswordRequirements.length > 0) {
+      notification.error({
+        message: (
+          <>
+            <div>
+              <div>Password must include:</div>
+              <ul>
+                <li>Capital letter</li>
+                <li>Special character</li>
+              </ul>
+              <div> Your password is missing: </div>
+              <br />
+            </div>
+            <ul style={{ margin: 0 }}>{missingPasswordRequirements}</ul>
+          </>
+        ),
+        icon: <ExclamationCircleTwoTone twoToneColor="#eb2f96" />,
+        placement: "top",
+        duration: 15,
+        style: { width: 350 },
+      });
+      return;
+    }
```

Comment:
Cool this feedback is really good! It looks pretty duplicated in the output though. You can just remove the top one or something
![duplicate-message](https://github.com/rknizzle/propermesh-view/assets/40612461/1bdf3406-83c8-439e-9f4f-d711caaf0edf)